src/org/etools/j1939_84/controllers/part01/Part01Step14Controller.java

```diff
@@ -104,10 +104,14 @@ protected void run() throws Throwable {
 
                      })
                      .forEach(dm26System -> {
-                         String moduleName = Lookup.getAddressName(dm26System.getSourceAddress());
-                         String systemName = dm26System.getName().trim();
-                         addFailure("6.1.14.2.a - " + moduleName + " response for a monitor " + systemName
-                                 + " in DM5 is reported as supported and is reported as complete/not supported DM26 response");
+                         if (getEngineModelYear() >= 2019 && dm26System.getId() == CompositeSystem.MISFIRE) {
```

Comment:
This would be better if you inverted the `if` expression or added a filter to the stream.